monkey/monkey_island/cc/app.py

```diff
@@ -1,14 +1,16 @@
 import os
 import re
-import uuid
-from datetime import timedelta
+from pathlib import Path
 from typing import Iterable, Set, Type
 
 import flask_restful
 from flask import Flask, Response, send_from_directory
+from flask_mongoengine import MongoEngine
+from flask_security import MongoEngineUserDatastore, Security
 from werkzeug.exceptions import NotFound
 
 from common import DIContainer
+from monkey_island.cc.models import Role, User
```

Comment:
I see that user contains email. Will we have to switch to [username](https://flask-security-too.readthedocs.io/en/stable/configuration.html#SECURITY_USERNAME_ENABLE)?